### PR TITLE
Feature/mvp instructor

### PR DESCRIPTION
**Pull request by alexbojko**
This PR includes:

SideBar:
- issue #302 

Courses:
- issue #296
- issue #297
- issue #298
- issue #299
- issue #301

Courslets:
- issue #303
- issue #304
- issue #305
- issue #310

Units:
- issue #306
- issue #307
- issue #311
- issue #315
 
Answers:
- issue #313
- issue #314

### CODE REVIEW

File: mysite/templates/ctms/courseunit_confirm_delete.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
```
Comment (maxsocl) [inline mysite/templates/ctms/courseunit_confirm_delete.html]: @alexbojko need to use the same name for all related files - not courseunit but courselet?

File: mysite/templates/ctms/courseunit_detail.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{% extends "base.html" %}
```
Comment (maxsocl) [inline mysite/templates/ctms/courseunit_detail.html]: @alexbojko same as above

File: mysite/templates/ctms/unit_form.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{% extends 'base.html' %}
```
Comment (maxsocl) [inline mysite/templates/ctms/unit_form.html]: @alexbojko same as above

File: mysite/templates/ctms/unitlesson_detail.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
+{% extends "base.html" %}
```
Comment (maxsocl) [inline mysite/templates/ctms/unitlesson_detail.html]: @alexbojko same as above - may be not unitlesson but unit?

File: mysite/templates/ctms/unitlesson_settings.html [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
```
Comment (maxsocl) [inline mysite/templates/ctms/unitlesson_settings.html]: @alexbojko same as above

File: mysite/ctms/admin.py [hunk 6]
```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+
+# Register your models here.
+from ctms.models import SharedCourse
+
+
+@admin.register(*[SharedCourse])
```
Comment (alexbojko) [inline mysite/ctms/admin.py]: because I thought that I will add more models here later

File: mysite/ctms/forms.py [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+class CourseForm(forms.ModelForm):
+    class Meta:
+        model = Course
+        fields = ('title', 'addedBy')
+        widgets = {
+            'addedBy': forms.HiddenInput
+        }
+
+class CourseUnitForm(forms.ModelForm):
```
Comment (alexbojko) [inline mysite/ctms/forms.py]: this form is no more needed. I delete it

**Comment by maxsocl** [comment]
@alexbojko check travis-ci build.

**Pull request merged**